report/renderer/performance-category-renderer.js

```diff
@@ -237,12 +237,8 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       // Performance diagnostics should only have score display modes of "informative" and "metricSavings"
       // If the score display mode is "metricSavings", the `score` will be a coarse approximation of the overall impact.
       // Therefore, it makes sense to sort audits by score first to ensure visual clarity with the score icons.
-      const scoreA = a.auditRef.result.scoreDisplayMode === 'informative'
-        ? 100
-        : Number(a.auditRef.result.score);
-      const scoreB = b.auditRef.result.scoreDisplayMode === 'informative'
-        ? 100
-        : Number(b.auditRef.result.score);
+      const scoreA = a.auditRef.result.score || 0;
+      const scoreB = b.auditRef.result.score || 0;
```

Comment:
I kinda like this solution because it means the informative audit score accurately communicates where in the sort order informative audits should be. Without this change we need a special exception for informative audits.